### PR TITLE
Adding Example for Affinity Rules

### PR DESCRIPTION
**Pull request by chrislovecnm**
- updated operator.yaml with feature gate for Affinity rules
- added example in example.yaml for affinity rules

### CODE REVIEW

File: config/templates/example.yaml.in [hunk 1]
```diff
@@ -38,7 +39,25 @@ spec:
...
 {{- /* .LatestStableCrdbVersion: latest stable version of CockroachDB */}}
+# You can set either a version of the db or a specific image name
 # cockroachDBVersion: {{ .LatestStableCrdbVersion }}
   image:
     name: cockroachdb/cockroach:{{ .LatestStableCrdbVersion }}
+  # nodes refers to the number of crdb pods that are created
+  # via the statefulset 
   nodes: 3
+  # affinity is a new API field that is behind a feature gate that is
+  # disabled by default.  To enable please enable, see operator.yaml
```
Comment (taroface) [inline config/templates/example.yaml.in:50]: `To enable please enable`

Some typos here

File: config/templates/example.yaml.in [hunk 2]
```diff
@@ -38,7 +39,25 @@ spec:
...
+  # The affinity field will accept any podSpec affinity rule.
+  #affinity:
```
Comment (taroface) [inline config/templates/example.yaml.in:52]: `#affinity:`

Could you offset this and everything below by 1 space, to be consistent with the rest of the commented blocks?

File: examples/example.yaml [hunk 3]
```diff
@@ -37,7 +38,25 @@ spec:
...
 # cockroachDBVersion: v21.1.5
   image:
     name: cockroachdb/cockroach:v21.1.5
+  # nodes refers to the number of crdb pods that are created
+  # via the statefulset 
   nodes: 3
+  # affinity is a new API field that is behind a feature gate that is
+  # disabled by default.  To enable please enable, see operator.yaml
+  # The affinity field will accept any podSpec affinity rule.
+  #affinity:
```
Comment (taroface) [inline examples/example.yaml:51]: Ditto

File: examples/example.yaml [hunk 4]
```diff
@@ -37,7 +38,25 @@ spec:
...
   tlsEnabled: true
+# You can set either a version of the db or a specific image name
 # cockroachDBVersion: v21.1.5
   image:
     name: cockroachdb/cockroach:v21.1.5
+  # nodes refers to the number of crdb pods that are created
+  # via the statefulset 
   nodes: 3
+  # affinity is a new API field that is behind a feature gate that is
+  # disabled by default.  To enable please enable, see operator.yaml
```
Comment (taroface) [inline examples/example.yaml:49]: `To enable please enable`

Ditto

**Review by taroface — CHANGES_REQUESTED** [review]

File: config/templates/example.yaml.in (hunk 2 above)
Comment (chrislovecnm) [reply config/templates/example.yaml.in:52]: > Could you offset this and everything below by 1 space, to be consistent with the rest of the commented blocks?

If we do this the user will need to delete the `#` and the extra space character as well.  It did it that way so they only delete the `#` character.

Comment (keith-mcclellan) [reply config/templates/example.yaml.in:54]: unfortunately all of our example yaml's are like this so we really should keep to the standard. I would rather change all of them but it'd be an effort to do that and doing it different ways in different files is extremely confusing

**Review by keith-mcclellan — CHANGES_REQUESTED** [review]
can we update the indentations on the commented out lines to meet the standard in our other examples please?

**Review by keith-mcclellan — APPROVED** [review]
LGTM

**Pull request merged**